lib/floe/workflow/runner/docker_mixin.rb

```diff
@@ -0,0 +1,18 @@
+module Floe
+  class Workflow
+    class Runner
+      module DockerMixin
+        def image_name(image)
+          image.match(%r{^(?<repository>.+/)?(?<image>.+):(?<tag>.+)$})&.named_captures&.dig("image")
+        end
+
+        def container_name(image)
+          name = image_name(image)
+          raise ArgumentError, "Invalid docker image [#{image}]" if name.nil?
+
+          "floe-#{name}-#{SecureRandom.uuid}"
```

Comment:
I think there's technically a prior bug here since the pod_name can only be 63 characters, and it's possible to have an image name longer than (63 - 2 - 4 - 36 =) 21 characters.  Before this change, we could afford 26 characters, which is still pretty small.

I don't think we need a full UUID either - thinking we might be able to use, say, 8 or 12 random hex bytes.